tracer/test/Datadog.Trace.Tests/DatabaseMonitoring/DatabaseMonitoringPropagatorTests.cs

```diff
@@ -79,16 +89,35 @@ public void ExpectedTagsInjected(string expectedComment, string env = null, stri
         [InlineData("/*dddbs='Test.Service%20%21%23%24%25%26%27%28%29%2A%2B%2C%2F%3A%3B%3D%3F%40%5B%5D-mysql',dde='te%23%27sti%2F%2Ang',ddps='Test.Service%20%21%23%24%25%26%27%28%29%2A%2B%2C%2F%3A%3B%3D%3F%40%5B%5D',ddpv='1.%2A0.0'*/", "Test.Service !#$%&'()*+,/:;=?@[]", "te#'sti/*ng", "1.*0.0")]
         public void ExpectedTagsEncoded(string expectedComment, string service, string env, string version)
         {
-            var span = _tracer.StartSpan(operationName: "mysql.query", parent: SpanContext.None, serviceName: $"{service}-mysql", traceId: (TraceId)7021887840877922076, spanId: 407003698947780173);
+            var span = _v0Tracer.StartSpan(operationName: "mysql.query", parent: SpanContext.None, serviceName: $"{service}-mysql", traceId: (TraceId)7021887840877922076, spanId: 407003698947780173);
             span.Context.TraceContext.Environment = env;
             span.Context.TraceContext.ServiceVersion = version;
             span.SetTraceSamplingPriority(SamplingPriority.AutoKeep);
 
-            var returnedComment = DatabaseMonitoringPropagator.PropagateSpanData(DbmPropagationLevel.Service, service, span.Context, IntegrationId.MySql, out bool traceParentInjected);
+            var returnedComment = DatabaseMonitoringPropagator.PropagateSpanData(DbmPropagationLevel.Service, service, span, IntegrationId.MySql, out bool traceParentInjected);
 
             // Always false since this test never runs for full mode
             traceParentInjected.Should().Be(false);
             returnedComment.Should().Be(expectedComment);
         }
+
+        [Theory]
+        [InlineData("Service", SamplingPriorityValues.AutoReject, "npgsql", "Test.Service-postgres", "/*dddbs='dbname',ddps='Test.Service'*/", false, "dbname")]
+        public void ExpectedCommentInjectedV1(string propagationMode, int? samplingPriority, string integration, string dbServiceName, string expectedComment, bool traceParentInjected, string dbName)
+        {
+            DbmPropagationLevel dbmPropagationLevel;
+            Enum.TryParse(propagationMode, true, out dbmPropagationLevel);
+
+            IntegrationId integrationId;
+            Enum.TryParse(integration, true, out integrationId);
+
+            var span = _v1Tracer.StartSpan(tags: new SqlV1Tags() { DbName = dbName }, operationName: "mysql.query", parent: SpanContext.None, serviceName: dbServiceName, traceId: (TraceId)7021887840877922076, spanId: 407003698947780173);
+            span.SetTraceSamplingPriority((SamplingPriority)samplingPriority.Value);
+
+            var returnedComment = DatabaseMonitoringPropagator.PropagateSpanData(dbmPropagationLevel, "Test.Service", span, integrationId, out var traceParentInjectedValue);
+
+            traceParentInjectedValue.Should().Be(traceParentInjected);
+            returnedComment.Should().Be(expectedComment);
+        }
```

Comment:
nit: if you're only testing a single set of values, make this a `[Fact]` and just declare the variables in the method:
```suggestion
        [Fact]
        public void ExpectedCommentInjectedV1()
        {
            var dbmPropagationLevel = DbmPropagationLevel.Service;
            var integrationId = IntegrationId.Npgsql;
            var samplingPriority = SamplingPriorityValues.AutoReject;
            var dbServiceName = "Test.Service-postgres";
            var expectedComment = "/*dddbs='dbname',ddps='Test.Service'*/";
            var traceParentInjected = false;
            var dbName = "dbname";

            var span = _v1Tracer.StartSpan(tags: new SqlV1Tags() { DbName = dbName }, operationName: "mysql.query", parent: SpanContext.None, serviceName: dbServiceName, traceId: (TraceId)7021887840877922076, spanId: 407003698947780173);
            span.SetTraceSamplingPriority((SamplingPriority)samplingPriority.Value);

            var returnedComment = DatabaseMonitoringPropagator.PropagateSpanData(dbmPropagationLevel, "Test.Service", span, integrationId, out var traceParentInjectedValue);

            traceParentInjectedValue.Should().Be(traceParentInjected);
            returnedComment.Should().Be(expectedComment);
        }
```